pkg/daemon/daemon.go

```diff
@@ -615,7 +615,7 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 }
 
 func (dn *Daemon) runLoginMonitor(stopCh <-chan struct{}, exitCh chan<- error) {
-	cmd := exec.Command("journalctl", "-b", "-f", "-o", "cat", "-u", logindUnit, "MESSAGE_ID="+sdMessageSessionStart)
+	cmd := exec.Command("journalctl", "-b", "-f", "-o", "cat", "-u", logindUnit)
```

Comment:
this effectively removes the filter for "New session .." entries and that's why we check them below. This seems to be working as grabbing the journal for the logind service _always_ has something (not sure if that's 100% valid all the time tho). Here's an example:

```
[root@ci-ln-t6mdh42-f76d1-9qqj4-master-1 /]# journalctl -f -o cat -b -u systemd-logind.service
Starting Login Service...
Watching system buttons on /dev/input/event0 (Power Button)
Watching system buttons on /dev/input/event1 (Sleep Button)
Watching system buttons on /dev/input/event2 (AT Translated Set 2 keyboard)
New seat seat0.
Started Login Service.
New session 1 of user core.
Session 1 logged out. Waiting for processes to exit.
Removed session 1.
New session 3 of user core.
Session 3 logged out. Waiting for processes to exit.
Removed session 3.
New session 5 of user core.
Session 5 logged out. Waiting for processes to exit.
Removed session 5.

```
So, the above is at least guaranteed to keep streaming messages and we can read them